index.html

```diff
@@ -0,0 +1,232 @@
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
+                     <button type="button" class="wrapper"><a href="#">See project</a></button>
+                 </article>
+             </article>
+             <article class="card-work-item">
+                 <article class="individual-card-item">
+                     <h3>Multi-Post Stories <br/> Gain+Glory</h3>
+                     <ul class="languages">
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
+                     <button type="button" class="wrapper"><a href="#">See project</a></button>
+                 </article>
+             </article>
+             <article class="card-work-item">
+                 <article class="individual-card-item">
+                     <h3>Multi-Post Stories <br/> Gain+Glory</h3>
+                     <ul class="languages">
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
+                     <button type="button" class="wrapper"><a href="#">See project</a></button>
+                 </article>
+             </article>
+             <article class="card-work-item">
+                 <article class="individual-card-item">
+                     <h3>Multi-Post Stories <br/> Gain+Glory</h3>
+                     <ul class="languages">
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
+                     <button type="button" class="wrapper"><a href="#">See project</a></button>
+                 </article>
+             </article>
+         </div>
+       </div>
+    </section>
+
+    <section class="about">
+        <div class="about-flex">
+            <div class="about-me">
+                <h2>About me</h2>
+                <p> I can help you build a product, feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded,
+                don't hestiate to contact me.</p>
+                <button type="button" class="wrapper"><a href="#">Get My Resume</a></button>
+            </div>
+            <div class="about-me-skils">
+                <div class="worklanguages">
+                    <div><img src="./images/icon-languages.svg" alt="Image depicting languages"></div>
+                    <div><h3>Languages</h3></div>
+                    <div class="lang-tags">
+                       <ul>
+                          <li>JavaScript</li>
```

Comment:
Can you kindly remove this black border-color you introduced to the button of your project

![image](https://github.com/zhorabay/My-portfolio/assets/20537712/08d098a9-e72b-4a33-abbd-f4c25d3ad0b9)